cli/src/commands/bisect/run.rs

```diff
@@ -72,6 +82,10 @@ pub(crate) struct BisectRunArgs {
     /// `$JJ_BISECT_TARGET` environment variable.
     #[arg(long, value_name = "COMMAND", required = true)]
     command: CommandNameAndArgs,
+    /// Whether to look for the first good revision or first bad revision.
+    /// Defaults to finding the first bad revision.
+    #[arg(long, value_name = "TARGET", value_enum, default_value_t = BisectionTarget::Bad)]
+    find_first: BisectionTarget,
```

Comment:
Did you consider making this a boolean flag called something like `--find-good` with a description like the following?

> Find the first good revision instead. Inverts the interpretation of exit statuses, so zero means bad and non-zero mean good.

Reply:
It felt more explicit to have the enum, but I'm fine with switching.